app/assets/stylesheets/fe_variables.scss

```diff
@@ -7,9 +7,32 @@ $size-6: 1rem;
 $size-6-5: 0.87rem;
 $size-7: 0.75rem;
 
-$afal-blue: #00b3ee;
-$white: #ffffff;
+// Colors
 $black: hsl(0, 0%, 4%);
+$black-bis: hsl(0, 0%, 7%);
+$black-ter: hsl(0, 0%, 14%);
+
+$grey-darker: hsl(0, 0%, 21%);
+$grey-dark: hsl(0, 0%, 29%);
+$grey: hsl(0, 0%, 48%);
+$grey-light: hsl(0, 0%, 71%);
+$grey-lighter: hsl(0, 0%, 86%);
+$grey-lightest: hsl(0, 0%, 93%);
+
+$white-ter: hsl(0, 0%, 96%);
+$white-bis: hsl(0, 0%, 98%);
+$white: hsl(0, 0%, 100%);
+
+$orange: hsl(14, 100%, 53%);
+$yellow: hsl(48, 100%, 67%);
+$green: hsl(141, 53%, 53%);
+$turquoise: hsl(171, 100%, 41%);
+// $cyan:         hsl(204, 71%,  53%);
+$blue: hsl(217, 71%, 53%);
+$purple: hsl(271, 100%, 71%);
+$red: hsl(348, 86%, 61%);
+
+$afal-blue: #00b3ee;
```

Comment:
Lets keep working with the Nomenclature of the web Colors... Use hsl(195, 100%, 47%) instead Hex